src/components/media/slideshow/SlideViewer.tsx

```diff
@@ -202,22 +203,27 @@ const InfoContainer = ({ currentImage, tagList, auth }: InfoContainerProps): Rea
           <TagList
             hovered
             list={tagList}
-            onDeleted={onTagDeletedHanlder}
+            onDeleted={onTagDeletedHandler}
             isAuthorized={isAuthorized}
             className='my-2'
           />}
       </div>
       {isAuthorized &&
         <div className='my-8'>
           <div className='text-primary text-sm'>Tag this climb</div>
-          <AddTag onTagAdded={onTagAddedHanlder} imageInfo={currentImage} className='my-2' />
+          <AddTag onTagAdded={onTagAddedHandler} imageInfo={currentImage} className='my-2' />
         </div>}
 
       {tagList?.length === 0 && isAuthorized &&
         <div className='my-8 text-secondary flex items-center space-x-1'>
           <LightBulbIcon className='w-6 h-6 stroke-1 stroke-ob-primary' />
           <span className='mt-1 text-xs'>Your tags help others learn more about the crag</span>
         </div>}
+
+      {isAuthorized &&
+        <div className='my-8'>
+          <RemoveImage imageInfo={currentImage} tagCount={tagList?.length} />
```

Comment:
Perhaps introduce a callback eg 'onImageDeleted()` in RemoveImage component.  The function is called after a successful delete.  You can handle the callback here in SliderViewer and call onClose() which will bubble the event up to the gallery component that will take care of closing the full screen viewer.